samples/snippets/sample_firewall.py

```diff
@@ -164,11 +166,11 @@ def delete_firewall_rule(project_id: str, firewall_rule_name: str):
         create_firewall_rule(default_project_id, rule_name)
         try:
             print("Rule created:")
-            print_firewall_rule(default_project_id, rule_name)
+            print(get_firewall_rule(default_project_id, rule_name))
```

Comment:
Do we publish `get_firewall_rule`? It seems we don't, but we also didn't publish the print method then, so not a regression.